projects/discussion-ui/src/lib/services/discuss-utils.service.ts

```diff
@@ -24,4 +24,13 @@ export class DiscussUtilsService {
     return 'rgba(255, 255, 255, 80%)';
   }
 
+  /**
+   * The htmlDecode() method parses a string containing either HTML or XML
+   * @param  {any} str
```

Comment:
Start adding the description for the parameters as well
Add example also. It helps to understand the code better.